recipe/meta.yaml

```
@@ -11,14 +11,16 @@ source:
   folder: source
 
 build:
-  number: 0
-  skip: true  # [win]
+  number: 1
+  skip: true  # [win and vc<14]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake
```

It's not related to this PR but this should have already been there instead relying on the host's `make`
```suggestion
    - cmake
    - make  # [unix]
```